chpl/chpl-service/src/main/java/gov/healthit/chpl/app/statistics/AsynchronousStatisticsInitializor.java

```diff
@@ -54,6 +45,16 @@
 		Future<Long> totalOpenNonConformities = asyncStats.getTotalOpenNonconformities(dateRange);
 		Future<Long> totalClosedNonConformities = asyncStats.getTotalClosedNonconformities(dateRange);
 		
+		if(stats instanceof CurrentStatistics){
+			Future<Long> totalActive2014Listings = asyncStats.getTotalActive2014Listings(dateRange);
+			Future<Long> totalActive2015Listings = asyncStats.getTotalActive2015Listings(dateRange);
+			Future<List<CertifiedBodyStatistics>> totalActiveListingsByCertifiedBody = asyncStats.getTotalActiveListingsByCertifiedBody(dateRange);
+			
```

Comment:
Should still break this if statement into two components. The first is for the `Future<Long> ... = asyncStats.get....` and the second is for the `currentStats.set....(total...get())`. The first section should go above with the other `Future<Long>` statements, and the second down in the following `currentStats...get()` section